spec.html

```diff
@@ -5254,6 +5254,18 @@ <h1>CreateDataPropertyOrThrow ( _O_, _P_, _V_ )</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-createnonenumerabledatapropertyorthrow">
+      <h1>CreateNonEnumerableDataPropertyOrThrow ( _O_, _P_, _V_ )</h1>
+      <p>The abstract operation CreateNonEnumerableDataPropertyOrThrow takes arguments _O_ (an Object), _P_ (a Property Key), and _V_ (an ECMAScript language value). It is used to create a new non-enumerable own property of an object. It throws a *TypeError* exception if the requested property update cannot be performed. It performs the following steps when called:</p>
+      <emu-alg>
+        1. Let _newDesc_ be the PropertyDescriptor { [[Value]]: _V_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+        1. Return ? _O_.[[DefineOwnProperty]](_P_, _newDesc_).
```

Comment:
```suggestion
        1. Return ? DefinePropertyOrThrow(_O_, _P_, _newDesc_).
```
that way it can reuse the existing machinery, and we don't add a new callsite for the internal methods.

Reply:
+1. I think the current PR is actually wrong, since [[DefineOwnProperty]] doesn't throw a TypeError when it returns false.